esmvaltool/preprocessor/_multimodel.py

```diff
@@ -73,7 +73,10 @@ def _compute_statistic(datas, name):
         # get all NOT fully masked data - u_data
         # datas is per time point
         # so we can safely NOT compute stats for single points
-        u_datas = [data for data in datas if not np.all(data.mask)]
+        if datas.ndim == 1:
```

Comment:
Do we still need all this and the cases below, now that we moved time-area after multi-model?

Reply:
Alright, let's leave it there :+1: